x-pack/plugins/synthetics/server/synthetics_service/formatters/browser.ts

```diff
@@ -5,20 +5,16 @@
  * 2.0.
  */
 
-import {
-  Formatter,
-  commonFormatters,
-  objectFormatter,
-  stringToObjectFormatter,
-  arrayFormatter,
-} from './common';
+import { browserFormatters as basicBrowserFormatters } from '../../../common/formatters/browser/formatters';
+import { Formatter, commonFormatters } from './common';
 import { BrowserFields, ConfigKey } from '../../../common/runtime_types/monitor_management';
 import { DEFAULT_BROWSER_ADVANCED_FIELDS } from '../../../common/constants/monitor_defaults';
 import { tlsFormatters } from './tls';
+import { arrayFormatter, objectFormatter, stringToObjectFormatter } from './formatting_utils';
 
 export type BrowserFormatMap = Record<keyof BrowserFields, Formatter>;
 
-const throttlingFormatter: Formatter = (fields) => {
+const throttlingFormatter: Formatter = (fields, key) => {
```

Comment:
Is key being used in this formatter? I see it added but not used.

Reply:
Or I guess if it's not being used
```suggestion
const throttlingFormatter: Formatter = (fields, _key) => {
```